examples/build.gradle

```diff
@@ -23,7 +23,7 @@ allprojects {
             maven { url 'https://jitpack.io' }
         }
         dependencies {
-            classpath 'com.android.tools.build:gradle:3.0.0-alpha4'
+            classpath 'com.android.tools.build:gradle:3.0.0-beta2'
```

Comment:
update android gradle together with others in a separated PR

Reply:
We already have https://github.com/realm/realm-java/pull/4899

Reply:
Any reason it cannot be merged?

Reply:
@cmelchior the reason is written in the description of the PR.

Reply:
And beta3 is adding far more critical issue...

Reply:
If you'll notice, this is utterly orthogonal. CI builds it with no problem.

Reply:
Reverted...